packages/vkui/src/components/Typography/EllipsisText/EllipsisText.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { RootComponentProps } from '../../RootComponent/RootComponent';
+import styles from './EllipsisText.module.css';
+
+export interface EllipsisTextProps extends RootComponentProps<HTMLElement> {
+  /**
+   * Пользовательская маскимальная ширина.
+   *
+   * Используйте в случаях, когда у контейнера ширина зависит от размера контента,
+   * другими словами, когда ширина не ограничена.
+   */
+  maxWidth?: number;
+}
+
+/** Компонент ограничивает текстовый контент убирая его в многоточие.
+ *
+ * @see https://vkcom.github.io/VKUI/#/EllipsisText
+ */
+const EllipsisText = ({ children, maxWidth, ...restProps }: EllipsisTextProps) => (
+  <span className={styles['EllipsisText']}>
+    <span style={{ maxWidth }} className={styles['EllipsisText__content']} {...restProps}>
```

Comment:
```suggestion
const EllipsisText = ({ children, maxWidth, className, getRootRef, ...restProps }: EllipsisTextProps) => (
  <span ref={getRootRef} className={classNames(styles['EllipsisText'], className)} {...restProps}>
    <span style={{ maxWidth }} className={styles['EllipsisText__content']}>
```

для `getRootRef` расширь пропы интерфейсом `HasRootRef` (см. примеры в др. компонентах)